README.md

```diff
@@ -24,4 +24,17 @@ You can build it continuously so that every time you make a change, build files
 npm run watch
 ```
 
-This is very useful when used together with [npm link ](https://docs.npmjs.com/cli/v7/commands/npm-link) so that, when you are working on another project that depends on this design system, you can make changes here and changes will be immediately availabe there.
\ No newline at end of file
+### Testing
+
+We can test two ways:
+
+**Storybook**
+
+For components that include storybooks, we can run `npm run storybook` to view the components in a sandbox-ed storybook environment.
+
+**npm link**
+
+1. Run `npm link` in your checkout of this repo.
+2. Spin up a local copy of [the VIP Dashboard](https://github.com/automattic/vip-ui) and navigate to a page using the linked components from `@automattic/vip-design-system`
```

Comment:
Minor comment

This repo is not necessarily coupled with the Dashboard, although it's the only repo using it AFAIK.

Reply:
Yeah, just figured it'd be useful to reference since that's where we'll use this system most heavily.